src/app/components/BibPage/BibDetails.jsx

```diff
@@ -144,14 +145,14 @@ const BibDetails = (props) => {
               `${fieldLabel} - ${bibValue.prefLabel}`,
             )
           }
-          dir={stringDirection(linkText)}
+          dir={stringDirection(linkText, useParallels)}
```

Comment:
Just curious: what's the reason to implement this as 

`dir={stringDirection(linkText, useParallels)}`

instead of

`dir={useParallels ? stringDirection(linkText) : null}`

i.e. why pass an argument to a function that effectively nullifies the function instead of simply not calling the function?

Reply:
That would work too. Originally I wanted to let the `stringDirection` method use context to determine if the parallels feature is enabled, so no argument would need to be passed. But I haven't figured out how to do that yet.